portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,23 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <div class="h1--container">
+        <h1>My Portfolio</h1>
+      </div>
       <p>This is my portfolio.</p>
+      <p>I love to learn whatever I can. From fitness, to economics, to software engineering, you have me on board</p>
+      <br/>
+      <h3>
+          Check out my portfolio below
+
+          <div class="upside-down">
+            <img src="https://img.webmd.com/dtmcms/live/webmd/consumer_assets/site_images/article_thumbnails/other/cat_relaxing_on_patio_other/1800x1200_cat_relaxing_on_patio_other.jpg">
+            </img>
+          </div>
+      </h3>
+      <iframe src="https://lev-stambler.github.io/Portfolio/"></iframe>
+      <p>Or go to my portfolio sit be clicking <a href="https://lev-stambler.github.io/Portfolio/">This link.</a> I made it using a custom frontend framework I created in 2018 named trivial.JS</p>
```

Comment:
s/sit/site  (substitute word "sit" with "site")
s/be/by
s/This/this (lowercase)